### PR TITLE
Update Test App - To create multiple concurrent tasks

### PR DESCRIPTION
**Pull request by shoatman**
- Added new StartActivity and made this the first activity launched by the test app
- StartActivity in turn has a button to launch a new instance of MainActivity
- MainActivity has android:documentLaunchMode="always" which results in a new task every time the button is pressed
- From the recents view once launch task is clicked twice.  You will see 2 separate tasks each containing an instance of MainActivity
- This is necessary to to manually test the multiple concurrent sign in support that Samsung has requested.

Document Launch Mode:
https://developer.android.com/guide/topics/manifest/activity-element#dlmode
https://www.youtube.com/watch?v=4Y3JMvbcxQE

### CODE REVIEW

File: testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/StartActivity.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+public class StartActivity extends AppCompatActivity {
+
+    //private static final String TAG = MainActivity.class.getSimpleName();
```
Comment (AdamBJohnsonx) [inline testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/StartActivity.java:37]:
```suggestion
    //private static final String TAG = StartActivity.class.getSimpleName();
```

File: testapps/testapp/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -56,9 +57,17 @@
             </intent-filter>
         </activity>
 
+        <activity
+            android:name="com.microsoft.identity.client.testapp.MainActivity"
+            android:theme="@style/AppTheme.NoActionBar"
+            android:windowSoftInputMode="stateHidden"
+            android:taskAffinity=""
+            android:documentLaunchMode="always">
+        </activity>
```
Comment (AdamBJohnsonx) [inline testapps/testapp/src/main/AndroidManifest.xml:66]: No intent filter here?

File: testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/StartActivity.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+            }
+        });
+
+    }
+
+    @Override
+    public View onCreateView(View parent, String name, Context context, AttributeSet attrs) {
+
+
```
Comment (AdamBJohnsonx) [inline testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/StartActivity.java:70]:
```suggestion
```

**Review by AdamBJohnsonx — APPROVED** [review]
This is just adding test app features to launch new activities.  Cool.

**Review by iamgusain — APPROVED** [review]
:shipit:

File: testapps/testapp/src/main/AndroidManifest.xml (hunk 2 above)
Comment (shoatman) [reply testapps/testapp/src/main/AndroidManifest.xml:66]: No.  It's just need for the activity that you want to be used by the launcher.

**Pull request merged**